bin/oref0-setup.sh

```diff
@@ -294,7 +294,7 @@ function move_mmtune () {
 
 function install_or_upgrade_nodejs () {
     # install/upgrade to latest node 8 if neither node 8 nor node 10+ LTS are installed
-    if ! nodejs --version | grep -e 'v8\.' -e 'v1[02468]\.' >/dev/null; then
+    if ! node --version | grep -e 'v8\.' -e 'v1[02468]\.' >/dev/null; then
         echo Installing node 8
         # Use nodesource setup script to add nodesource repository to sources.list.d
         sudo bash -c "curl -sL https://deb.nodesource.com/setup_8.x | bash -" || die "Couldn't setup node 8"
```

Comment:
Should this also use n?

Reply:
It don't see why it couldn't. I don't have an edison board so I was a bit hesitant to change things that I couldn't test directly. In theory, this part could be entirely replaced by lines 318-331 of this same file (checking that n is installed, doing so if not, and then installing node).

Reply:
Confirmed it will install via n, but only v8 on the edison.

Reply:
Latest changes should work on Edison. I would appreciate a re-test and an update as to what version is installed.